src/ims/store/_db.py

```diff
@@ -1859,7 +1859,6 @@ def setFieldReportAttribute(txn: Transaction) -> None:
 
         self._log.info(
             "{author} updated field report #{fieldReportNumber}: {attribute}={value}",
-            storeWriteClass=FieldReport,
```

Comment:
this was causing a second, undesirable SSE to be fired with each Field Report update